### DISCUSSION TITLE
Verbindung zwischen Lucene- und Detailsuche (GIS-Suche)

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#76] [Q&A]
Hypothese: Die im \etc\search\xxx.json hinterlegte Suche identifiziert den darunterliegenden Geo-Datensatz.
Die Annahme dabei ist, dass der Primärschlüssel "fid" heißt.
Für das LVG wäre es hilfreich wenn auch ein Primärschlüssel "ogc_fid" unterstützt würde.
Der Grund: "ogc_fid" ist der Standard Primärschlüssel von OGC Daten und in allen Daten des Landes Vorarlberg der Primärschlüssel.

**Comment by gis-eni** [comment]
Um die Verbindung von lucene zu deinen Daten herzustellen, wird das objectID Feld verwendet. 
Mach deine Spalte zur objectID...

Den Vorteil sehe ich zwar nicht, wir alles nur intern verwendet?!

Also eigentlich kein Grund das zu ändern
